packages/core/src/languages/solidity/predeploys.ts

```diff
@@ -175,21 +221,39 @@ export const deploySphinxSystem = async (
   for (const relayer of relayers) {
     const chainId = (await provider.getNetwork()).chainId
 
+    const currentDripVersion = fetchDripVersionForNetwork(chainId)
+    const baseDripName = `sphinx_fund_${relayer}`
+    const dripName =
+      baseDripName + (currentDripVersion > 0 ? `_${currentDripVersion}` : '')
```

Comment:
Yeah, this is intentional; it's because the first version I deployed did not include a version number. This logic just replicates that so we get a consistent setup on all networks.